portal/views/portal.py

```diff
@@ -595,10 +595,34 @@ def home():
 def admin():
     """user admin view function"""
     # can't do list comprehension in template - prepopulate a 'rolelist'
-    users = User.query.filter_by(deleted=None).all()
+
+    request_org_list = request.args.get('org_list', None)
+
+    if request_org_list:
+        org_list = set()
+
```

Comment:
This block looks familiar.  Are we doing this elsewhere?  Is a refactor of common code in order?